added troubleshooting tips to private networking doc (#1476)
superfly/docs: networking/private-networking.html.md

#### Remove a tunnel

To remove a tunnel, run `fly wireguard remove`. You can specify the organization and tunnel name on the command line or be prompted for both.

### Troubleshooting

> Having trouble connecting to a Fly.io hosted resource? The Private Network VPN feature makes it easy for a local machine to establish a fast, reliable connection to a Fly.io organization. However, when you can't connect to something, it's helpful to establish a baseline of what is, or what is not, working.

Reviewer: Suggest re-wording this slightly. Maybe just remove the second sentence and the "however" from the third sentence? We don't need to sell the feature here and we'll usually try to avoid saying something is "easy" (at least in straight-up tech docs and especially in troubleshooting docs).

#### Am I connected to Fly.io VPN?

When connected locally, you can run a `dig` command to list all the apps your connection has access to.

```cmd
dig _apps.internal TXT +short
```
```output
my-app,my-app-db
```

If results are returned, you have a VPN connection to an org at Fly.io and the results list all the app names.

If no results are returned, either you do not have a WireGuard VPN connection open or there are NO apps running in the org.

#### Am I connected to the right Fly.io org?

WireGuard connections are created to a specific org. Each org's network is isolated from other orgs. You may have a VPN connection to Fly.io, but it may be to a different org than where your app is located. By default, you are working with your **personal** organization. If the app is in a business focused org or a shared org, you need to ensure your VPN connection is the org where the app lives.

> We can also try and ping our app's machine.

Reviewer: ```suggestion
You can also try to ping your app's Machine.
```
